packages/teams-js/src/public/video.ts

```diff
@@ -5,10 +5,12 @@ import { errorNotSupportedOnPlatform, FrameContexts } from './constants';
 import { runtime } from './runtime';
 
 /**
+ * @hidden
```

Comment:
Are these APIs still under development? If so and there are plans to continue changing them before "releasing" them, consider adding a "@beta" tag to them as well, indicating to developers that they might change and they should not build production level apps using these APIs without understanding that they might break at any time.

Reply:
Good point. Added the tag.

Reply:
See my other comment after further thought about already releasing these APIs.